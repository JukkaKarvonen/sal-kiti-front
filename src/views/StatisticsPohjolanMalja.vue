<template>
  <div>
    <b-row>
      <b-col>
        <h1>{{ $t("statistics.statistics") }}</h1>
      </b-col>
    </b-row>
    <StatisticsPohjolanMalja />
  </div>
</template>

<script>
/**
 * Pohjolan malja statistics view
 */
import StatisticsPohjolanMalja from "@/components/StatisticsPohjolanMalja.vue";

export default {
  name: "StatisticsPohjolanMaljaView",
  components: {
    StatisticsPohjolanMalja
  },
  created() {
    document.title =
      this.$t("title.statistics") + " | " + this.$t("title.prefix");
  }
};
</script>

<style scoped></style>
