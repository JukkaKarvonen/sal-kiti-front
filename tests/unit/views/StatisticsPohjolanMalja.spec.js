import { shallowMount, createLocalVue } from "@vue/test-utils";
import StatisticsPohjolanMalja from "@/views/StatisticsPohjolanMalja.vue";

import BootstrapVue from "bootstrap-vue";

const localVue = createLocalVue();
localVue.use(BootstrapVue);

describe("StatisticsPohjolanMalja.vue", () => {
  it("renders content", () => {
    const wrapper = shallowMount(StatisticsPohjolanMalja, {
      localVue
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
