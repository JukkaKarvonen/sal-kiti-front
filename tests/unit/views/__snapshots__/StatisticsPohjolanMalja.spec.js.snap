// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatisticsPohjolanMalja.vue renders content 1`] = `
<div>
  <b-row-stub
    tag="div"
  >
    <b-col-stub
      tag="div"
    >
      <h1>
        statistics.statistics
      </h1>
    </b-col-stub>
  </b-row-stub>
   
  <statisticspohjolanmalja-stub />
</div>
`;
